server/config/routes.js

```diff
@@ -83,7 +83,7 @@ exports.configure = function (app) {
   winston.debug('Configuring routes');
 
   // exposed to the outside without authentication
-  app.get('/languages', users.getLanguages);
+  app.get('/languages', users.languages);
```

Comment:
We discussed as a team, and we decided to use verb for API exposed methods.
Here is an example : 

```
app.post('/creditor_groups', creditorGroups.create); //the verb to create
app.get('/creditor_groups', creditorGroups.list); //the verb to list
app.get('/creditor_groups/:uuid', creditorGroups.detail); //the verb to detail
app.put('/creditor_groups/:uuid', creditorGroups.update); //the verb to update
```
to languages is not a verb.